packages/java/plugin-base/src/main/java/dev/hilla/plugin/base/HillaAppInitUtility.java

```diff
@@ -15,15 +15,15 @@ public class HillaAppInitUtility {
     private static final Logger LOGGER = LoggerFactory
             .getLogger(HillaAppInitUtility.class);
 
-    private static final String REACT_SKELETON = "https://github.com/vaadin/skeleton-starter-hilla-react/archive/refs/heads/v2.1.zip";
+    private static final String REACT_SKELETON = "https://github.com/vaadin/skeleton-starter-hilla-react/archive/refs/heads/v2.5.zip";
```

Comment:
The `v2.5` branch was only used during the pre-releasing phase. I suggest using the default `v2` branch, which gets regularly updated. 

```suggestion
    private static final String REACT_SKELETON = "https://github.com/vaadin/skeleton-starter-hilla-react/archive/refs/heads/v2.zip";
```